src/TTGamesExplorerRebirthUI/Forms/DATForm.cs

```diff
@@ -24,7 +24,58 @@ public DATForm(string gameFolderPath, string datFilePath)
             _gameFolderPath = gameFolderPath;
             _datFilePath = datFilePath;
             _datArchive = new(datFilePath);
+            {
```

Comment:
The `{}` isn't needed here

Reply:
Scope to seperate the chunky loading form code from the rest of the code, I'd be alright to remove the brackets if you feel it's needed